build_parameters.json

```diff
@@ -6,8 +6,8 @@
 		{
 			"ACTION": "COMPILE",
 			"TYPE": "POM",
-			"FOLDER": "step-distribution-parent/",
-			"BEFORE": "step-os-upgrade-distribution",
+			"FOLDER": ".",
+			"BEFORE": "step-os-upgrade-distribution-new",
```

Comment:
I guess we need to keep both plans for older branches, may be we can find a better name to make it more clear in the build server?

Reply:
Plan updated and working with both layouts now.